packages/runtime/test/transport/identityDeletionPeerProcessing.test.ts

```diff
@@ -51,6 +51,8 @@ describe("IdentityDeletionProcess", () => {
 
     test("peer should be notified about cancelled deletion process", async function () {
         await services1.transport.identityDeletionProcesses.initiateIdentityDeletionProcess();
+        await syncUntilHasEvent(services2, PeerToBeDeletedEvent, (e) => e.data.id === relationshipId);
```

Comment:
please add a comment, this should not be necessary and should be removed as soon the backbone resolves the core issue.